.github/CODEOWNERS

```diff
@@ -110,6 +110,7 @@ apps/public-docsite-v9 @microsoft/cxe-red @microsoft/cxe-coastal @microsoft/flue
 apps/theming-designer @microsoft/fluentui-react
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 45 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26332&commitId=f8ea9b9b0088355733ede17e7994899c317c6ed4&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1730 screenshots unchanged, 0 screenshots with different dimensions and 45 screenshots with visible difference.


  <details><summary><b>unknown</b> 45 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CheckboxField converged.fieldLabel.default.chromium.png|1313| Changed|
|ComboboxField converged.base.default.chromium.png|1410| Changed|
|ComboboxField converged.hint.default.chromium.png|1541| Changed|
|ComboboxField converged.required.default.chromium.png|1410| Changed|
|ComboboxField converged.size-large.default.chromium.png|1663| Changed|
|ComboboxField converged.size-small.default.chromium.png|1406| Changed|
|ComboboxField converged.validation.default.chromium.png|8241| Changed|
|InputField converged.base.default.chromium.png|1314| Changed|
|InputField converged.hint.default.chromium.png|1445| Changed|
|InputField converged.required.default.chromium.png|1314| Changed|
|InputField converged.size-large.default.chromium.png|1587| Changed|
|InputField converged.size-small.default.chromium.png|1262| Changed|
|InputField converged.validation.default.chromium.png|7815| Changed|
|ProgressField converged.base.default.chromium.png|1709| Changed|
|ProgressField converged.hint.default.chromium.png|1840| Changed|
|ProgressField converged.required.default.chromium.png|1709| Changed|
|ProgressField converged.validation.default.chromium.png|9158| Changed|
|RadioGroupField converged.base.default.chromium.png|1530| Changed|
|RadioGroupField converged.hint.default.chromium.png|1661| Changed|
|RadioGroupField converged.required.default.chromium.png|1530| Changed|
|RadioGroupField converged.validation.default.chromium.png|9326| Changed|
|SelectField converged.base.default.chromium.png|1381| Changed|
|SelectField converged.hint.default.chromium.png|1512| Changed|
|SelectField converged.required.default.chromium.png|1381| Changed|
|SelectField converged.size-large.default.chromium.png|1695| Changed|
|SelectField converged.size-small.default.chromium.png|1299| Changed|
|SelectField converged.validation.default.chromium.png|8318| Changed|
|SliderField converged.base.default.chromium.png|1290| Changed|
|SliderField converged.hint.default.chromium.png|1421| Changed|
|SliderField converged.required.default.chromium.png|1290| Changed|
|SliderField converged.validation.default.chromium.png|8488| Changed|
|SpinButtonField converged.base.default.chromium.png|1383| Changed|
|SpinButtonField converged.hint.default.chromium.png|1514| Changed|
|SpinButtonField converged.required.default.chromium.png|1383| Changed|
|SpinButtonField converged.validation.default.chromium.png|8136| Changed|
|SwitchField converged.base.default.chromium.png|1126| Changed|
|SwitchField converged.hint.default.chromium.png|1257| Changed|
|SwitchField converged.required.default.chromium.png|1126| Changed|
|SwitchField converged.validation.default.chromium.png|7095| Changed|
|TextareaField converged.base.default.chromium.png|1324| Changed|
|TextareaField converged.hint.default.chromium.png|1455| Changed|
|TextareaField converged.required.default.chromium.png|1324| Changed|
|TextareaField converged.size-large.default.chromium.png|1581| Changed|
|TextareaField converged.size-small.default.chromium.png|1273| Changed|
|TextareaField converged.validation.default.chromium.png|7858| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>